lambda/getBinInterval.ts

```diff
@@ -15,14 +16,13 @@ export const getBinInterval =
 		durationHours: number
```

Comment:
**problem:** use the type `HistoricalDataTimeSpan` here so you ensure that these property names align across the codebase (and it's clear where they are coming from)